apps/protocol-frontend/src/components/ContributionTypesPie.tsx

```diff
@@ -0,0 +1,129 @@
+import { Flex, Heading, Text, useBreakpointValue } from '@chakra-ui/react';
+import { ResponsivePie } from '@nivo/pie';
+import { GovrnSpinner } from '@govrn/protocol-ui';
+
+type ContributionTypesCount = {
+  activity_id: number;
+  activity_name: string;
+  count: number;
+};
+interface ContributionTypesPieProps {
+  contributionActivityData: ContributionTypesCount[] | undefined;
+  isFetching: boolean;
+  isLoading: boolean;
+  isError: boolean;
+}
+
+const brandColorMap = [
+  '#ffb4e2',
+  '#fb84ce',
+  '#f854ba',
+  '#f526a6',
+  '#db0f8d',
+  '#ab076d',
+  '#76024e',
+];
```

Comment:
Do we have a central place to put our css color palette? Are these the came on the bar chart

Reply:
Ah, I was thinking about this -- we have these in our `theme.primary` object in the theme, but this was a convenient mapping of those values in a format we need for these graphs. I'm thinking we could add this to the theme somewhere and export as a constant from there incase we want to update it -- that way it's still in 1 place.

Alternatively we could map from the theme palette itself, that way at least that's the source of truth. I think I may have had that originally for the heat map and removed it. Great call out though @alexkeating -- we should do 1 or the other so we can keep this easily updatable